src/decisionengine/framework/engine/Workers.py

```diff
@@ -98,13 +99,13 @@ def run(self):
 
```

Comment:
In the configuration of the log rotation, the `f"{myname}"` should be replaced by `CHANNELLOGGERNAME`, the logger name changed, and for the file name you can simply use the variable name `f"{myfilename}"` becomes `myfilename`.
This is code that was not in the last change

Reply:
- `f"{myname}"` is not the same as `CHANNELLOGGERNAME`
- I don't think we should change the name of the logger - see earlier comment
- changed `f"{myfilename}"` to `myfilename`.

Reply:
About 1, I know, I think this is a bug: `f{myname}` or `myname` is the name of the channel (e.g. AWSCalculation), in all the python modules and here the logger is retrieved with `getLogger(CHANNELLOGGERNAME)` (which is "channellog"). The configuration will not apply to that logger if you use a different name. Using the same name is the correct thing to do. The loggers will be different because they are in different processes.

Reply:
I don't think there is a bug. `myname` is the name of the **handler** for a particular handler corresponding to the logger in the channel (of the same name) with the name `CHANNELLOGGERNAME`. By keeping the handler names  distinct, we have more flexibility - we can format the handlers in the logging_configDict if we choose.